appservice/src/main/java/nu/yona/server/messaging/rest/MessageController.java

```diff
@@ -103,6 +103,19 @@
 		}
 	}
 
+	private void TeasingSonarQube()
+	{
+		try
+		{
+			final String password = "Secret";
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Remove this hard-coded password. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://build.dev.yona.nu:9000/coding_rules#rule_key=squid%3AS2068)